step2/src/utils/@common/localStorage.ts

```diff
@@ -0,0 +1,21 @@
+import { STORAGE_KEY } from '../../constants/key';
+import { ERROR_MESSAGE } from '../../constants/messages';
+import { RestaurantType } from '../../types/restaurants';
+
+export const saveStorage = (data: RestaurantType[]) => {
+  try {
+    localStorage.setItem(STORAGE_KEY, JSON.stringify(data));
+  } catch (error) {
+    console.error(ERROR_MESSAGE.SAVE_LOCAL_STORAGE);
+  }
+};
+
+export const getStorage = (): RestaurantType[] | null => {
+  try {
+    const data = localStorage.getItem(STORAGE_KEY);
+    return data && JSON.parse(data);
```

Comment:
&& 연산자를 이렇게 활용할 수도 있겠지만 정확한 반환값이 명시적으로 드러나지 않아 코드를 읽는 입장에서는 의도가 분명해보이지 않을 수 있습니다.

```suggestion
if (data) {
  return JSON.parse(data);
}

return null;
```
리턴값을 좀 더 분명하게 명시하는 게 좋을 것이라 생각합니다.

Reply:
아하 그렇군요! 
줄일 수 있는 코드는 줄이자는 생각을 버리고, 코드를 읽는 사람의 입장을 더욱 생각하는 습관을 길러야겠습니다...

8262be17dd6f1ea033e2d37b3535dcd6f14a3deb